docs/core_docs/docs/modules/data_connection/retrievers/index.mdx

```diff
@@ -98,45 +98,3 @@ Let's walk through what's happening here.
 4. We ask questions!
 
 See the individual sections for deeper dives on specific retrievers.
```

Comment:
link custom retriever page here in case people linked to the custom retriever section